src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -442,6 +447,10 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('use_subdomain', help='Specify whether to use indirect CNAME validation.',
                    arg_type=get_enum_type(['true', 'false']))
         c.argument('tags', tags_type, default=None)
+        c.argument('enable_sftp', arg_type=get_three_state_flag(), min_api='2021-08-01',
+                   is_preview=True, help='Enable Secure File Transfer Protocol.')
+        c.argument('enable_local_user', arg_type=get_three_state_flag(), min_api='2021-08-01',
+                   is_preview=True, help='Enable local user features.')
```

Comment:
```suggestion
        c.argument('enable_sftp', arg_type=get_three_state_flag(), min_api='2021-08-01',
                   help='Enable Secure File Transfer Protocol.')
        c.argument('enable_local_user', arg_type=get_three_state_flag(), min_api='2021-08-01',
                   help='Enable local user features.')
```
Service is requesting for GA SFTP